docs/include/conf/virtual/writing-day-project-faq.rst

```diff
@@ -0,0 +1,129 @@
+technologies, sharpen their skills, and add to their portfolios. The world of open source has a seemingly endless 
+number of wonderful open source projects. Documentarians want to write the docs, your open source project docs. 
+
+It's a match made at Write the Docs!
+
+Reminder, Write the Docs is a ticketed conference and you need to purchase a ticket to attend Writing Day.
+
+How does Writing Day work?
+--------------------------
+
+Great question, here is an overview:
+
+* First the event coordinator welcomes everyone and gives a Writing Day introduction.
+* During the introduction, we invite people to give a project summary.
+* The project introduction include the title, a short description, and their table number - so you can find them.
+* After all the projects are announced, you are welcome to check them out!
+
+Attendees can stay for as long or short as they want! They can try out as many projects as they feel like. 
+Some folks enjoy sticking at one table for the entire event, others like to table hop and get a feel for 
+multiple projects. 
+
+As a project, you may not accomplish all of your goals or agenda. This is normal for events like these, m
+any projects continue to receive contributions after Writing Day ends.
+
+Get involved with Writing Day
+------------------------------
+
+If you are someone looking to bring a project to Writing Day, this information is for you.
+
+Here is how you can get involved and maximize your experience:
+
+* Submit your project and we'll add it to our :doc:`/conf/{{shortcode}}/{{year}}/writing-day/` project list. You can
+  fill out the Writing Day project form or submit a PR to the Writing Day page.
+* Prepare your project to maximize your experience.
+
+While it is perfectly valid to show up to Writing Day, tell us about your project, and ask for volunteers during the event; 
+there are benefits to submitting it in advance.
+
+Potential benefits vary based on when you submit your project to Writing Day, they can include:
+
+* Free advertisement - your project info is available on our conference site project list for all visitors to find.
+* Social media promotion - we post about Writing Day and the cool projects that are submitted.
+* Increased views and attendance - the earlier you submit your project, the more visitors and attendees are likely to 
+  see it and see it more than once. Making them more likely to seek you out during the event.
+  
+We have seen that the more projects we have available for attendees to potentially contribute to, they more likely
+they are to attend to learn more. It benefits us, them, and you!
+
+Need help convincing your OSS project community or corporate benefactors on the benefits of attending Writing Day? 
+See :ref:`making-writing-day-case`.
+
+Submit your project
+^^^^^^^^^^^^^^^^^^^
+
+There is a basic level of information we need to add your project to the Writing Day list. `Submitting the form <>`_ is an
```

Comment:
@rosewms would you mind creating and linking the form here so the build doesn't break?